hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/DFSConfigKeys.java

```diff
@@ -1557,6 +1557,12 @@
   public static final double
       DFS_DATANODE_RESERVE_FOR_ARCHIVE_DEFAULT_PERCENTAGE_DEFAULT = 0.0;
 
+  public static final String
```

Comment:
Shall we include "disk-tiering" (or some other words) in the names for these several configurations related to the disk-tiering feature?

Reply:
Good idea